css/style.css

```diff
@@ -0,0 +1,117 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+body {
```

Comment:
[OPTIONAL] Great job but there is an overflowing issue coming from the default body margin: 8px. If you remove them, that's perfection 👨‍🎨
<img width="1116" alt="Screenshot 2023-01-12 at 2 05 28 AM" src="https://user-images.githubusercontent.com/25887253/211938285-099b5788-4cda-4462-be8d-c6db644d510d.png">

Reply:
Hi @huseyinbiyik 

Thank you for your informative feedback!

The my overflow problem still persist even though I made both margin and padding to 0 for the body. You mentioned "if you remove them" Can you suggest me specifically what I am going to remove?

Reply:
Hey @ezraermy ,
There is no overflowing issue after deleting the default body margin,. Check the gif below.
![Screen Recording 2023-01-12 at 10 20 51 PM](https://user-images.githubusercontent.com/25887253/212162468-3b3f9e7d-d1c0-438c-92da-cbed61792aac.gif)